nitro-protocol/gas-benchmarks/fixtures.ts

```diff
@@ -498,70 +498,3 @@ export async function challengeVirtualPaymentChannelWithVoucher(
     gasUsed,
   };
 }
-
-export async function respondWithChallengeVirtualPaymentApp(
```

Comment:
Yup, we can definitely do that! It just requires rewriting some test helpers (the existing challenge helper is coupled to challenging on turnnum=1) and it was more work than I wanted to tackle 😛 

We could probably spend some time refactoring some of these test helpers to make them a little less brittle as well. Instead of having a challenge function that sets up a specific state with vouchers, we could separate them out into two functions so it's easy to challenge with any state.